spec/integration/cas/client_spec.rb

```diff
@@ -1,48 +1,26 @@
-# encoding: utf-8
+# frozen_string_literal: true
+
 RSpec.describe 'Cassette::Client, Cassette::Authentication integration' do
   shared_examples_for 'a Cassette client and validator' do
-    let(:config) { fail 'implement config!' }
+    let(:config) { raise 'implement config!' }
     let(:client) { Cassette::Client.new(config: config) }
     let(:authentication) { Cassette::Authentication.new(config: config) }
 
     it 'generates an ST' do
       expect(client.st_for(config.service)).not_to be_blank
     end
 
-    it 'validates an ST, extracting the user' do
+    context 'when validates an ST, extracting the user' do
       st = client.st_for(config.service)
       user = authentication.ticket_user(st, config.service)
 
-      expect(user).not_to be_blank
-      expect(user.login).to eql(config.username)
-    end
-  end
-
-  context 'with a configuration' do
-    # it_behaves_like "a Cassette client and validator" do
-    #   let(:config) do
-    #     OpenStruct.new(
-    #       username: "test.user",
-    #       password: "anothersecret",
-    #       base: "https://anothercas.example.org",
-    #       service: "qualquercoisa"
-    #       base_authority: "SYSTEM"
-    #     )
-    #   end
-    # end
-  end
+      it do
```

Comment:
Could you merge these two `it`s, naming as
```
it 'returns the configured username' do
```

Reply:
Maybe this won't be possible due to rubocop - it may be checking for "one `expect` per `it`" rule. But I'm favorable for adding a description for each `it`. What about 

```ruby
it "returns an user object` do
#...
end 

it "instantiates the returned user with the correct login information" do
#...
end
```